torchgeo/models/unet.py

```diff
@@ -0,0 +1,153 @@
+        meta={
+            'dataset': 'FTW',
+            'in_chans': 8,
+            'num_classes': 2,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b3',
+            'publication': 'https://arxiv.org/abs/2409.16252',
+            'repo': 'https://github.com/fieldsoftheworld/ftw-baselines',
+            'bands': _ftw_sentinel2_bands,
+            'license': 'CC-BY-4.0',
+        },
+    )
+    SENTINEL2_3CLASS_FTW = Weights(
+        url='https://huggingface.co/torchgeo/ftw/resolve/d2fdab6ea9d9cd38b491292cc9a5c8642533cef5/commercial/3-class/sentinel2_unet_effb3-5d591cbb.pth',
+        transforms=_ftw_transforms,
+        meta={
+            'dataset': 'FTW',
+            'in_chans': 8,
+            'num_classes': 3,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b3',
+            'publication': 'https://arxiv.org/abs/2409.16252',
+            'repo': 'https://github.com/fieldsoftheworld/ftw-baselines',
+            'bands': _ftw_sentinel2_bands,
+            'license': 'CC-BY-4.0',
+        },
+    )
+    SENTINEL2_2CLASS_NC_FTW = Weights(
+        url='https://huggingface.co/torchgeo/ftw/resolve/d2fdab6ea9d9cd38b491292cc9a5c8642533cef5/noncommercial/2-class/sentinel2_unet_effb3-bf010a31.pth',
+        transforms=_ftw_transforms,
+        meta={
+            'dataset': 'FTW',
+            'in_chans': 8,
+            'num_classes': 2,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b3',
+            'publication': 'https://arxiv.org/abs/2409.16252',
+            'repo': 'https://github.com/fieldsoftheworld/ftw-baselines',
+            'bands': _ftw_sentinel2_bands,
+            'license': 'non-commercial',
+        },
+    )
+    SENTINEL2_3CLASS_NC_FTW = Weights(
+        url='https://huggingface.co/torchgeo/ftw/resolve/d2fdab6ea9d9cd38b491292cc9a5c8642533cef5/noncommercial/3-class/sentinel2_unet_effb3-ed36f465.pth',
+        transforms=_ftw_transforms,
+        meta={
+            'dataset': 'FTW',
+            'in_chans': 8,
+            'num_classes': 3,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b3',
+            'publication': 'https://arxiv.org/abs/2409.16252',
+            'repo': 'https://github.com/fieldsoftheworld/ftw-baselines',
+            'bands': _ftw_sentinel2_bands,
+            'license': 'non-commercial',
+        },
+    )
+
+
+def unet(
+    weights: Unet_Weights | None = None,
+    classes: int | None = None,
+    *args: Any,
+    **kwargs: Any,
+) -> Unet:
+    """U-Net model.
+
+    If you use this model in your research, please cite the following paper:
+
+    * https://arxiv.org/abs/1505.04597
+
+    .. versionadded:: 0.8
+
+    Args:
+        weights: Pre-trained model weights to use.
+        classes: Number of output classes. If not specified, the number of
+            classes will be inferred from the weights.
+        *args: Additional arguments to pass to :func:`segmentation_models_pytorch.create_model`
```

Comment:
```suggestion
        *args: Additional arguments to pass to ``segmentation_models_pytorch.create_model``
```
Then we don't have to ignore the lack of docs. Alternatively, could try to add docs.